vignettes/estimate_static_severity.Rmd

```diff
@@ -40,8 +38,12 @@ distribution. The methods used here originate from @nishiura2009.
 This vignette demonstrates how to implement the core functions within
 _cfr_. Specifically, we will cover
 
-1. Why naive estimates of CFR or HFR generated during an ongoing outbreak 
-require adjusting, given out of sync time series are often being compared. 
+::: {.alert .alert-warning}
+New to calculating disease severity using _cfr_? You might want to see the ["Get started" vignette first](cfr.html).
```

Comment:
I like this idea of a get started vignette, is it the one currently entitled "cfr"? could we name it "Get started with `cfr`"?

Reply:
Thanks - so this is due to how {pkgdown} presents vignettes on the website. The vignette with the same name as the package appears on the top navigation bar as "Get started". So the vignette file name here has to be `cfr.Rmd`, and it is referred to in other vignettes as `cfr.html`.
<img width="1141" alt="Screenshot 2023-07-31 at 12 01 24" src="https://github.com/epiverse-trace/cfr/assets/25272594/7918aa24-1cd4-42c2-90f3-0bfecf11f075">